fork-visualizer/src/constants.js

```diff
@@ -0,0 +1,51 @@
+export const ENABLE_GRAPH_VIEW = false
+export const ENABLE_TEST_VIEW = false
+
+// used for the info overlay for latest block number (not working)
+export const ENABLE_BLOCK_COUNTER = false
+// used for the info overlay for block info (not working)
+export const ENABLE_BLOCK_LIST = false
+export const ENABLE_MINIMAP = false
+
+export const MAX_CHAIN_COUNT = 100
+export const AUTO_SCROLL_DEFAULT = false
+
+export const INITIAL_WS_ADDRESSES = [
+    "ws://100.109.138.126:9944",
+    "ws://100.109.138.126:8844",
+    "ws://100.109.138.126:7744",
+    "ws://100.109.138.126:6644",
+    "ws://100.109.138.126:5544",
```

Comment:
Why there is a need for init addresses?

Reply:
It is a convenience we liked when testing this out. We were starting the UI over and over and it got old adding all the same ones. I kind of want to keep this (or kevcoxe talked about wiring it up to an environment variable).

I think the value should just be `[ ws://localhost:9944 ]` though or something.